### PR TITLE
docs(core): Update inject error documentation

### PR DESCRIPTION
**Pull request by atscott**
The inject function is also available when using EnvironmentInjector.runInContext.

### CODE REVIEW

**Review by alxhub — APPROVED** [review]

**Review by jessicajaniuk — APPROVED** [review]
reviewed-for: public-api

File: aio/content/errors/NG0203.md [hunk 1]
```diff
@@ -1,9 +1,10 @@
 @name `inject()` must be called from an injection context
 @category runtime
-@shortDescription `inject()` must be called from an injection context (a constructor, a factory function or a field initializer)
+@shortDescription `inject()` must be called from an injection context (a constructor, a factory function, a field initializer, or a function used with `EnvironmentInjector#runInContext`)
```
Comment (AndrewKushnir) [inline aio/content/errors/NG0203.md:3]: super nit: should we consider making it a bit less "strict" and list possible contexts as examples, not an exhaustive list? We may expand the support for `inject` function in other places in the future (like Router guards) and this content might get outdated.
```suggestion
@shortDescription `inject()` must be called from an injection context, such as a constructor, a factory function, a field initializer, or a function called from the `EnvironmentInjector#runInContext` callback.
```

**Review by AndrewKushnir — APPROVED** [review]

**Review by dylhunn — APPROVED** [review]

File: aio/content/errors/NG0203.md (hunk 1 above)
Comment (atscott) [reply aio/content/errors/NG0203.md:3]: "or a function called from the `EnvironmentInjector#runInContext` callback"
and 
"a function used with `EnvironmentInjector#runInContext`"

are saying the same thing, in my opinion. Router guards would be "used with `EnvironmentInjector#runInContext`" under the hood and I don't think changing the wording makes it less strict. I have, however updated the sentence to remove the parentheses.

Comment (AndrewKushnir) [reply aio/content/errors/NG0203.md]: Thanks Andrew 👍

**Review by pkozlowski-opensource — APPROVED** [review]
LGTM

Reviewed-for: public-api 
Reviewed-for: fw-core

**Comment by AndrewKushnir** [comment]
This PR was merged into the repository by commit 9e3994c7cc3813069ca09e7d4e287d79803b286d.

**Pull request closed without merging**